<launch>
  <arg name="view_only" default="true"/>

  <node pkg="robot_state_publisher" exec="robot_state_publisher">
      <param name="robot_description" value="$(command '$(exec-in-pkg xacro xacro) $(find-pkg-share teleop_tasks)/urdf/rightHand.urdf.xacro')" />
      <param name="frame_prefix" value="right_hand/"/>
  </node>

  <group if="$(eval '\'$(var view_only)\' == \'false\'')">
    <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share teleop_tasks)/config/right_hand_config.rviz"/>
  </group>

  <group if="$(eval '\'$(var view_only)\' == \'true\'')">
    <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share teleop_tasks)/config/hand_config.rviz"/>
  </group>

  <node pkg="joint_state_publisher_gui" exec="joint_state_publisher_gui"/>
</launch>